sherpa-onnx/csrc/hypothesis.h

```diff
@@ -60,10 +60,10 @@ struct Hypothesis {
   std::string Key() const {
     // TODO(fangjun): Use a hash function?
     std::ostringstream os;
-    std::string sep = "-";
```

Comment:
please change it to
```
std::string sep;
```
and leave the code in the for loop unchanged.

Reply:
@csukuangfj I changed the code as suggested, which looks like the following

```cpp
  std::string Key() const {
    // TODO(fangjun): Use a hash function?
    std::ostringstream os;
    std::string sep;
    for (auto i : ys) {
      os << i << sep;
      sep = "-";
    }
    return os.str();
  }
```

For `0, 0, 120`, the key is `"00-120-"`. I assume `0-0-120` is the correct join result. Let me know if I miss anything. Thanks.

Reply:
Please change
```
os << i << sep;
```
to
```
os << sep << i;
```

Thanks!

Reply:
@csukuangfj  Just changed as suggested. Seem working well. Thanks a lot!